frontend/packages/client/src/components/Community/CommunityEditorLinks.js

```diff
@@ -164,16 +68,16 @@ export default function CommunityEditorLinks(props = {}) {
       </div>
       <div
         style={{ position: "relative" }}
-        className="is-flex is-align-items-center mt-2"
+        className="is-flex is-align-items-center mt-4"
       >
         <input
           type="text"
           name="twitter"
           className="rounded-sm border-light py-3 pr-3  column is-full"
-          value={links?.twitterUrl}
+          value={fields["twitterUrl"]}
           maxLength={200}
           onChange={(event) =>
-            setLinks((state) => ({ ...state, twitterUrl: event.target.value }))
+            onChangeHandler["twitterUrl"](event.target.value)
```

Comment:
```suggestion
            onChangeHandler("twitterUrl")(event.target.value)
```

Reply:
I think we could also dry this up a little bit by creating a new prop for the row since they all seem to be the same. Then you could map over an array of objects to to output each row. Let me know if you need additional clarity on this happy to pair with you.

Reply:
yes! I like the approach, I created a configuration object that is used to render each one of the fields, also handles the icon it should be using...and passing this configuration as a default to the component as a prop. This adds more flexibility for adding new fields.